.github/actions/build-gstreamer/action.yml

```
@@ -37,150 +30,32 @@ inputs:
   gst-plugins-rs-ref:
     description: 'Ref/tag to use for gst-plugins-rs'
     required: false
-  force:
-    description: 'Whether to force the build'
-    required: false
-  cleanup:
-    description: 'Whether to clean the Cerbero build directory'
-    required: false
-    default: 'true'
   no-cache:
     description: 'Whether to disable restoring from cache'
     required: false
     default: 'false'
-outputs:
   gstreamer-version:
     description: 'GStreamer version'
-    value: ${{ steps.version-info.outputs.gstreamer-version }}
-  cerbero-short-sha:
-    description: 'Short SHA of the Cerbero checkout'
-    value: ${{ steps.version-info.outputs.cerbero-short-sha }}
-  cerbero-path:
-    description: 'Path to the Cerbero checkout'
-    value: ${{ inputs.cleanup != 'true' && 'cerbero' }}
+    required: true
+    default: ''
+outputs:
```

Outputting the GStreamer version here is useful, as an input, it isn't used at all.

```suggestion
outputs:
  gstreamer-version:
    description: 'GStreamer version built'
    value: ${{ steps.cerbero-config.outputs.version }}
```